package.json

```diff
@@ -5,18 +5,23 @@
   "directories": {
     "lib": "lib"
   },
+  "bin": "./bin/sonarjs",
   "scripts": {
     "test": "jest",
     "clear": "rm -rf lib/*",
-    "build": "yarn clear && yarn && tsc -p ."
+    "build-mvn": "cd sonarmini && mvn clean install && cd ..",
+    "copy-jars":
+      "cp sonarmini/target/sonarjs-cli-*-SNAPSHOT.jar lib/ && mkdir lib/plugins && cp sonarmini/target/plugins/* lib/plugins/",
```

Comment:
@inverno imo jar should have another name

Reply:
`miniscanner`

Reply:
@inverno `mini-scanner`?